src/lucky_record/errors.cr

```diff
@@ -0,0 +1,20 @@
+# = Lucky Record Errors
+#
+# Generic Lucky Record exception class.
+class LuckyRecordError < Exception
+end
+
+# Raised when Lucky Record cannot find a record by given id
+class RecordNotFound < LuckyRecordError
```

Comment:
Could you nest this in a `LuckyRecord` module? `class LuckyRecord::RecordNotFoundError`

Oh and it should end with `Error`. I think that's the Crystal convention

Reply:
It would be better only `LuckyRecord::RecordNotFoundError `or `LuckyRecord::Errors::RecordNotFoundError`?

Im wrapping the classes with the module instead of each line.